README.md

```diff
@@ -4,3 +4,116 @@
 # MQTT bridge for Apache Kafka®
 
 This project provides a software component which acts as a bridge between [MQTT 3.1.1](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html) and an [Apache Kafka®](https://kafka.apache.org/) cluster.
+
+## Running the bridge
+
+### Locally
+
+Clone the repository and update your dependencies using maven.
+Afterward, edit the `config/application.properties` file which contains the configuration. And provide a valid [topic mapping rules file]().
+
+Once your configuration is ready, start the bridge using:
+
+```bash
+<application> --config-file config/application.properties --mapping-rules=path/to/mapping-rules
```

Comment:
There is a placeholder  name, <application>,  because we don't know what the final application name will be?

Reply:
Yeah this line doesn't have much sense right now. We should package the application and provide a jar which is runnable, so that we can replace `<application>` with something meanigful.

Reply:
@ppatierno should we package the application now?

Reply:
Right now we don't have an application which does what it's aim for ... so would not make sense to package. 
At the same time, this line doesn't make sense.
Right now, maybe, we could avoid the section about running the bridge and adding it later when it will be possible to run. @kyguy what's your taken on it?

Reply:
for me sounds good removing it.